### DISCUSSION TITLE
Use TShock with the default game rules?

### DISCUSSION THREAD

**Discussion by ToddGeorgeKelly** [Pryaxis/TShock#2375] [Q&A] [answered]
There seems to be a massive amount of things that are restricted to the default group and even when i add them as permissions they still dont seem to be work. I can only assume i am doing permissions all wrong. 

I mostly wanted to use TShock for the extra commands on the server's end and I wanted the game to run just like normal but I am having a hard time trying to figure out how to run the server and just have the 'normal' permissions be applied to my friends that join. Like letting them use wormhole potions, boss summoning items, etc. 

Could someone help me out? Thank you in advance for your patience. I love what TShock can do!

**Comment by bartico6** [comment, marked as answer]
[This is what you're looking for](https://github.com/Pryaxis/Vanillafier)
